server/db.go

```diff
@@ -89,6 +91,11 @@ func DbConnect(ctx context.Context, logger *zap.Logger, config Config) (*sql.DB,
 	if err = db.QueryRowContext(pingCtx, "SELECT version()").Scan(&dbVersion); err != nil {
 		logger.Fatal("Error querying database version", zap.Error(err))
 	}
+	if strings.Split(dbVersion, " ")[0] == "CockroachDB" {
```

Comment:
I suggest we invert the logic slightly. Assume we should use the existing `ExecuteInTx` logic *unless* the version string does not contain `"CockroachDB"`. This way our default remain the current logic which is "safe", and we try to use the new logic conditionally on top.

Reply:
Scratch that, it works out to the same thing.